yb-voyager/src/query/queryparser/helpers_protomsg.go

```diff
@@ -422,15 +438,15 @@ func ProtoAsCTENode(msg protoreflect.Message) (*pg_query.CommonTableExpr, error)
 	return cteNode, nil
 }
 
-func ProtoAsDefElemNode(msg protoreflect.Message) (*pg_query.DefElem, error) {
+func ProtoAsDefElemNode(msg protoreflect.Message) (*pg_query.DefElem, bool) {
```

Comment:
I see a a mix of return patterns here for the `ProtoAs` methods. (returning bool vs return err). 
Is there a reason we might prefer one over the other?
Overall, we can pick any I think , but it would be good to be consistent. 
Can you pls make them consistent at least for the new ones that you're adding? I

Reply:
Yeah its consistent for the new ones or older ones i am reusing.

Some old code is consistent.

Reply:
I didn't get this. 
I still see some inconsistency? 
For example, 
- `ProtoAsIndexElemNode(msg protoreflect.Message) (*pg_query.IndexElem, error)`
- `ProtoAsColumnDef(msg protoreflect.Message) (*pg_query.ColumnDef, bool)`

Let's try to make this consistent?

Reply:
oh ok, my bad. Let me make it consistent for new code.